runtime/common/src/impl_moonbeam_xcm_call.rs

```diff
@@ -1,48 +0,0 @@
-// Copyright 2019-2022 PureStake Inc.
-// This file is part of Moonbeam.
-
-// Moonbeam is free software: you can redistribute it and/or modify
-// it under the terms of the GNU General Public License as published by
-// the Free Software Foundation, either version 3 of the License, or
-// (at your option) any later version.
-
-// Moonbeam is distributed in the hope that it will be useful,
-// but WITHOUT ANY WARRANTY; without even the implied warranty of
-// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-// GNU General Public License for more details.
-
-// You should have received a copy of the GNU General Public License
-// along with Moonbeam.  If not, see <http://www.gnu.org/licenses/>.
-
-#[macro_export]
-macro_rules! impl_moonbeam_xcm_call {
-	{} => {
-
-		pub struct MoonbeamCall;
-		impl CallDispatcher<Call> for MoonbeamCall {
```

Comment:
cc @girazoki Does this look right? I don't see `CallDispatcher` in upstream's Config...

Reply:
This should be in (you need to bring CallDispatcher to our polkadot fork). But I think you already did that no?

Reply:
I just finished restoring it, so this is resolved now.